Adam/hello-world.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+//
+// This is only a SKELETON file for the "Hello World" exercise.
+// It's been provided as a convenience to get you started writing code faster.
+//
+
+function helloWorld($name = null)
+{
+  if ($name === null){
+    return "Hello, World!";
+  }
+  else {
+    return helloWorldWithName($name);
+  }
+}
+function helloWorldWithName($name) {
+  return sprintf("Hello, %s!", $name);
```

Comment:
I'm just testing it this not final ... I just wanted to understand the
sprintf !!

I will submit my solution after french class 😂

On Monday, 11 July 2016, Yvan Sraka <notifications@github.com> wrote:

> In Adam/hello-world.php
> <https://github.com/simplonco/php-hello-world/pull/2#discussion_r70253664>
> :
>
> > +//
> > +// This is only a SKELETON file for the "Hello World" exercise.
> > +// It's been provided as a convenience to get you started writing code faster.
> > +//
> > +
> > +function helloWorld($name = null)
> > +{
> > +  if ($name === null){
> > +    return "Hello, World!";
> > +  }
> > +  else {
> > +    return helloWorldWithName($name);
> > +  }
> > +}
> > +function helloWorldWithName($name) {
> > +  return sprintf("Hello, %s!", $name);
>
> Why sprintf 😮?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/simplonco/php-hello-world/pull/2/files/8a5e11621f279851746aec97d2f75ef7b52b8fab#r70253664>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AQ6ViliTvB107jZu5UxLpJ_WPgSMrfMQks5qUkC6gaJpZM4JJQqG>
> .
>